test/ts-scenario/steps/lib/deprecatedSDK.ts

```diff
@@ -132,6 +136,7 @@ export async function sdk_chaincode_install_for_org(ccType: 'golang' | 'car' | '
  * @param {Object} policy The endorsement policy object from the configuration file.
  * @return {Promise} The return promise.
  */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Maybe should define an explicit type in code to fix the lint error rathern than disable the rule